db/migrate/20200509090837_create_posts.rb

```diff
@@ -0,0 +1,20 @@
+class CreatePosts < ActiveRecord::Migration[6.0]
+ def self.up
+   create_table :peeps do |t|
+     t.integer :user_id
+     t.text :body
+     t.timestamps
+   end
+
+   create_table :users do |t|
+     t.string :username
+     t.text :fullname
+     t.timestamps
+   end
+ end
+
+ def self.down
+   drop_table :peeps
+   drop_table :users
+ end
+end
```

Comment:
Again, I love how this simplifies the CRUD process, definetley going to read up on this!